app/views/offers/show.html.erb

```diff
@@ -1,15 +1,16 @@
 <% if @offer.organization == current_organization %>
-  <p class="actions text-right">
+<div class="row">
+  <p class="col-xs-12 actions text-right">
     <% if admin? or @offer.user == current_user %>
       <%= render 'shared/post_actions', post: @offer %>
     <% end %>
     <% if current_user and @offer.user != current_user %>
       <%= link_to new_transfer_path(id: @offer.user.id, offer: @offer.id, destination_account_id: @destination_account.id),
-                  class: "btn btn-success" do %>
-        <%= glyph :time %>
+                  class: "mdc-time mdc-button mdc-button--raised" do %>        
```

Comment:
If you're using BEM this should be a modifier class of the mdc-button pattern, so I would do something like `class: "mdc-button mdc-button--raised mdc-button--time" do %>`.